docs/api-conventions.md

```diff
@@ -85,7 +89,7 @@ Every object SHOULD have the following metadata in a nested object field called
 
 Labels are intended for organizational purposes by end users (select the pods that match this label query). Annotations enable third-party automation and tooling to decorate objects with additional metadata for their own use.
 
-#### Spec and Status
+####<a name="specandstatus"></a>Spec and Status
```

Comment:
Is there a reason you added these anchors?  Markdown generates an anchor in a predictable format (this one would be `spec-and-status`).